dts/bindings/display/ilitek,ili9163.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2022, Tavish Naruka <tavishnaruka@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: ILI9163 132x162 display controller
+
+compatible: "ilitek,ili9163"
+
+include: ilitek,ili9xxx-common.yaml
+
+properties:
+    gamset:
+      type: uint8-array
+      default: [0x04]
```

Comment:
All use of defaults must be justified in the description. Please read https://docs.zephyrproject.org/latest/guides/dts/bindings.html#rules-for-mainline-bindings and update this PR accordingly.